src/gap.c

```diff
@@ -2998,12 +2998,20 @@ static Int InitLibrary (
     /* create windows command buffer                                       */
     WindowCmdString = NEW_STRING( 1000 );
 
+    UInt isvar = GVarName( "IsHPCGAP" );
+
 #ifdef HPCGAP
     UInt var = GVarName( "HPCGAP" );
     AssGVar( var, True );
-    MakeReadOnlyGVar( var );
+    MakeConstantGVar( var );
```

Comment:
Perhaps `IsBound(CONST_VAR)` should be optimized `true` if `CONST_VAR` is bound during coding? (I am not asking for this to be done in this PR, just wondering)